Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -216,6 +216,15 @@ class PruneCoreSymbolicationdCacheIfTooLarge(shell.ShellCommand):
                "/System/Library/Caches/com.apple.coresymbolicationd"]
 
 
+class PruneCoreSymbolicationdDataIfTooLarge(shell.ShellCommand):
+    name = "prune-coresymbolicationd-data-if-too-large"
+    description = ["pruning coresymbolicationd cache to < 10GB"]
+    descriptionDone = ["pruned coresymbolicationd cache"]
+    flunkOnFailure = False
+    haltOnFailure = False
+    command = ["python3", "Tools/Scripts/delete-if-too-large",
+               "/System/Volumes/Data/System/Library/Caches/com.apple.coresymbolicationd"]
```

Comment:
Running the same step/script twice just to pass two different arguments doesn't seems right. Either the script should accept multiple parameters and handle them appropriately, or the paths can be hard-coded inside the script itself.

Reply:
I agree. 

> Related, I implemented the most naive solution to this problem. I'm not sure how to best go about parameterising PruneCoreSymbolicationdCacheIfTooLarge within our use of buildbot.

How can I go about parameterising PruneCoreSymbolicationdCacheIfTooLarge? It's not clear to me if we can actually  (or should be )pass arguments or if the way we compose out list of steps is meant to be purely declarative.

Reply:
You can modify delete-if-too-large script so that users can pass multiple directories. You can either use argument parser (like https://docs.python.org/dev/library/argparse.html) or for this simple user-case, you can simply loop over sys.argv and store the arguments in a list of directories. Then just loop over that list and execute your logic to delete.

If the list of directories is going to be fixed, other option is to hard-code these directories names inside the script.